digitalforms-api/src/main/java/ca/bc/gov/open/pssg/rsbc/digitalforms/service/ApplicationFormServiceImpl.java

```diff
@@ -0,0 +1,77 @@
+package ca.bc.gov.open.pssg.rsbc.digitalforms.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import ca.bc.gov.open.pssg.rsbc.digitalforms.model.ApplicationFormData;
+import ca.bc.gov.open.pssg.rsbc.digitalforms.ordsclient.api.model.DigitalFormPatchRequest;
+import ca.bc.gov.open.pssg.rsbc.digitalforms.ordsclient.api.model.DigitalFormPostRequest;
+import ca.bc.gov.open.pssg.rsbc.digitalforms.ordsclient.application.ApplicationResponse;
+import ca.bc.gov.open.pssg.rsbc.digitalforms.ordsclient.application.ApplicationService;
+
+/**
+ * 
+ * Application form Service operations Implementation
+ * 
+ * @author sivakaruna
+ *
+ */
+@Service
+public class ApplicationFormServiceImpl implements ApplicationFormService {
+
+	@Autowired
+	private ApplicationService applicationService;
+
+	@Override
+	public ApplicationResponse getApplicationForm(String formType, String formGuid) {
+		return applicationService.getApplication(formGuid);
+	}
+	
+	@Override
+	public ApplicationResponse postApplicationForm(String formType, ApplicationFormData formData) {
```

Comment:
again, form type is not used BUT, this will be used in the future so let's leave it for now. Thanks.